extensions/blocks/videopress/editor.js

```diff
@@ -22,12 +51,24 @@ const addVideoPressSupport = ( settings, name ) => {
 	}
 
 	const { attributes, deprecated, edit, save, supports, transforms } = settings;
-
 	const { available, unavailableReason } = getJetpackExtensionAvailability( 'videopress' );
 
+	// Check if VideoPress is unavailable and filter the mediaplaceholder to limit options
+	if ( isSimpleSite && [ 'missing_plan', 'unknown' ].includes( unavailableReason ) ) {
+		addFilter( 'editor.MediaPlaceholder', 'jetpack/videopress', videoPressNoPlanMediaPlaceholder );
+		addFilter(
+			'editor.BlockListBlock',
+			`jetpack/videopress-with-has-warning-is-interactive-class-names`,
+			withHasWarningIsInteractiveClassNames( `core/video` )
+		);
+	}
+
 	// We customize the video block even if VideoPress it not available so we can support videos that were uploaded to
 	// VideoPress if it was available in the past (i.e. before a plan downgrade).
-	if ( available || [ 'missing_plan', 'missing_module' ].includes( unavailableReason ) ) {
+	if (
+		available ||
+		[ 'missing_plan', 'missing_module', 'unknown' ].includes( unavailableReason )
```

Comment:
IE11 doesn't support the array .includes method - not sure if we have a polyfill for that not.

Reply:
but i see that there are lots of uses of .includes in jetpack codebase already so I assume it isn't a problem 😃

Reply:
Yes, it was actually already being used in this file before I edited it, so I'm assuming it's okay. :)